lib/src/io.dart

```diff
@@ -24,6 +24,9 @@ import 'utils.dart';
 
 export 'package:http/http.dart' show ByteStream;
 
+/// Whether we're using Windows.
+final isWindows = Platform.operatingSystem == "windows";
```

Comment:
Is there a reason not to use `Platform.isWindows`?

Reply:
Nope, I just forgot it existed. Done!